docs/how-to/service-placement.rst

```diff
@@ -1,71 +1,94 @@
-Start Ceph services on any node of the MicroCeph cluster
-========================================================
+===================================
+Enable additional service instances
+===================================
 
-MicroCeph will automatically start Mon, Mgr, and Mds services on the first three deployed nodes in the cluster. Users who wish to enable additional instances of these services for scalability or resilience can do so by utilizing the enable command and specifying a node via the --target parameter.
+To ensure a base level of resiliency, MicroCeph will always try to enable a
+sufficient number of instances for certain services in the cluster. This
+number is set to three by default.
 
-.. list-table:: Supported Ceph Services
-   :widths: 30 70
-   :header-rows: 1
+These services affected by this include:
 
-   * - Name
-     - Description
-   * - mon
-     - Ceph Monitor Service
-   * - mgr
-     - Ceph Manager Service
-   * - mds
-     - Ceph Metadata Daemon Service
-   * - rgw
-     - Ceph Radosgw Service
+* MON (`Monitor service`_)
+* MDS (`Metadata service`_)
+* MGR (`Manager service`_)
+* RGW (`RADOS Gateway service`_)
```

Comment:
This sounds like MicroCeph will enable 3x RGW services by default, but that is not the case. MON, MDS and MGRs will be started on the first 3 nodes, but RGW always needs to be enabled by the operator

Reply:
You're right. Will adjust.